examples/air-purifier-app/air-purifier-common/pics/Activated Carbon Filter Monitoring Cluster Test Plan.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
Is this typical to have these XMLs inside the SDK repo? How are they used and should then not be in CSG repo?

Could we have names without spacese for nicer auto-complete (so I do not need to add quotes in command lines)?